packages/ynap-web-app/package.json

```diff
@@ -23,7 +23,7 @@
     "styled-components": "4.4.1",
     "ts-node": "8.10.2",
     "typescript": "3.9.7",
-    "ynap-parsers": "^1.15.0"
+    "ynap-parsers": "^1.15.1"
```

Comment:
Following https://lerna.js.org/docs/getting-started#package-dependency-management, we probably should not use `lerna boostrap` anymore. The `--production` flag has been removed, too.

I went with using `yarn add ../ynap-bank2ynab-converter` and `yarn install` for local testing, but that breaks the imports since it then requires the `src` directory to be in the import. (I'm no TS/JS expert, so this might be easy to solve in another way.

I see two ways here:

1. Using a local install in the GitHub actions if that's possible (see above)
2. Bump versions for separate packages only in separate PRs so they can be published separately and don't break tests for the others

I would much prefer the first solution if I'm honest.

Reply:
@leolabs Do you have any ideas/input for this?

Reply:
I'd also go for the first one. Using workspaces in Yarn like they show in the docs sounds like a good option!

Reply:
I'll check if we can switch to yarn workspaces easily.